src/encoded/schemas/treatment_agent.json

```diff
@@ -81,7 +81,7 @@
             "title": "Concentration units",
             "type": "string",
             "lookup": 40,
-            "enum": [
+            "suggest_enum": [
```

Comment:
doesn't this need to be "suggested_enum"?

Reply:
Of course it does - this is exactly why we do reviews even for small changes like this.  Doh! Thanks. Fixed now I hope.